ctd/read.py

```diff
@@ -448,7 +448,9 @@ def from_cnv(fname):
             )
             df.index.name = prkey
 
-    name = _basename(fname)[1]
+    if 'name' not in metadata:
```

Comment:
Just to make the linter happy.

```suggestion
    if "name" not in metadata:
```